### DISCUSSION TITLE
Miniconda download permission denied

### DISCUSSION THREAD

**Discussion by jbair34** [idaholab/moose#18196] [Q&A Getting Started] [answered]
Hi All,
I am trying to figure out how to install moose on a Windows machine and was able to get everything to work until it says to go to the Conda install. When I try to download miniconda I get the error below:

jbair34@DESKTOP-UHJNGJR:/mnt/c/Windows/system32$ sudo curl -L -O https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
[sudo] password for jbair34:
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Warning: Failed to create the file Miniconda3-latest-Linux-x86_64.sh:
Warning: Permission denied
  0 89.8M    0   900    0     0   2795      0  9:21:55 --:--:--  9:21:55  2795
curl: (23) Failed writing body (0 != 900)

I tried it without the sudo command as well with the same result. Any idea why it won't download?
Thanks,
Jake

**Comment by GiudGiud** [comment, marked as answer]
MOOSE is not natively supported on windows right now, you should try using the Windows Subsystem for Linux (WSL). 
See the instructions there
https://mooseframework.inl.gov/getting_started/installation/windows10.html

Regardless, 
I doubt you want to install conda in system32. You can install conda in like programs or your home directory.

**Comment by jbair34** [comment]
Wasn't even paying attention to what folder i was in... That System 32 thing was the issue. Thanks I got it going in WSL.
Jake
